src/lib/helpers.ts

```diff
@@ -0,0 +1,22 @@
+import { BASE_PATH } from "@src/consts";
+
+export const parseHref = (href: string | URL | undefined | null) => {
+  if (href) {
```

Comment:
Nitpick but you can invert this if, so that there is less indenting:
`  if (!href) return "";`